src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -94,6 +94,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
         type={BaseChatItemTypeProps.MY_ANON_CHAT}
         status={item?.status as ChatStatus}
         chatType={type}
+        chatItem={item}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `chatItem` prop has been added to the `BaseChatItem` component. Ensure that this prop is properly validated and handled in the `BaseChatItem` component to prevent potential issues with undefined or unexpected values.
```diff
+ chatItem={item}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->